workspace.json

```diff
@@ -2,6 +2,7 @@
   "version": 2,
   "projects": {
     "api": "apps/engineering",
-    "shared-server": "libs/server"
+    "shared-server": "libs/server",
+    "frontend": "apps/frontend"
```

Comment:
I had planned to add a proper project.json to the project itself, but run out of time. I also left the dependencies in their own package.json, instead of in the root one.